Fly_Swatter/radar.py

```diff
@@ -16,10 +16,10 @@ def __init__(self, phi, theta, r, time_spot):
   def tell(self):
     return (f"Phi: {self.phi}  " + f"Theta: {self.theta} " + f"Rho: {self.r}")
 
-def target_radar_sight(return_data):
-  # need a function that captures a location of the target
-  time = time.time()
-  return phi, theta, r, time
+# def target_radar_sight(return_data):
```

Comment:
this is worthy of beign commeneted out as it wasn't used and will probably be later implemeted in the mechanical pr